pegasus/sites.v3/code.org/views/header.haml

```diff
@@ -97,6 +97,13 @@
   var headerSetup = false;
 
   $ = jQuery;
+
+  $.ajaxSetup({
+    beforeSend: function(xhr) {
+      xhr.setRequestHeader('X_CSRF_TOKEN', '#{csrf_token}');
```

Comment:
The cached pages don't hit the CSRF protected teacher dashboard actions, and setting this elsewhere should be harmless.  However, I agree, I should look for a less global place to include this.  I will do that.